### PR TITLE
Document possible incompatibility

### LINKED ISSUE #364: There is no active transaction
PHP version: 8.0.6
MySQL version: 5.7

After updating php from 7.0 to 8.0 this issue arises. All the fixtures successfully load their data into database, but after that process fails.

image

I think this is related to PHP's BC break:
>  PDO::inTransaction() now reports the actual transaction state of the connection, rather than an approximation maintained by PDO. If a query that is subject to "implicit commit" is executed, PDO::inTransaction() will subsequently return false, as a transaction is no longer active.

### PR DESCRIPTION
**Pull request by greg0ire**
People are getting confused when getting "There is no active
transaction" from MySQL.

Closes #364

### CODE REVIEW

**Comment by greg0ire** [comment]
@rtek please review.

File: README.md [hunk 1]
```diff
@@ -190,6 +190,25 @@ class MyOtherFixture extends AbstractFixture
...
 
+## Atomicity
+
+This package exposes an `AbstractExecutor` class and 3 concrete implementations
+for `doctrine/orm`, `doctrine/mongodb-odm` and `doctrine/phpcr-odm`.
+
+The executors purge the database, then load the fixtures. The ORM
+implementation wraps these two steps in a database transaction, that way the
+loading either succeeds or fails cleanly, meaning nothing is actually changed
+in the database at the end of the loading. It delegates the purging to a
```
Comment (rtek) [inline README.md:201]: >that way the loading either succeeds or fails cleanly, meaning nothing is actually changed in the database ~at the end of the loading~ if the load fails.

**Review by rtek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (greg0ire) [reply README.md:201]: Do you mean "if the loading fails"?

Comment (rkorczykowski) [reply README.md:201]: yessir other than that its good

Comment (rtek) [reply README.md:201]: yessir other than that its good.

**Review by derrabus — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (greg0ire) [reply README.md:201]: Woops forgot to push the fix :man_facepalming: … that will be for another PR

Comment (greg0ire) [reply README.md:201]: https://github.com/doctrine/data-fixtures/pull/381

**Pull request merged**